packages/shared/components/gv-selection/gv-switch/index.tsx

```diff
@@ -60,13 +48,28 @@ class GVSwitch extends React.Component<GVSwitchProps> {
       className,
       color,
       value,
+      label,
       touched,
+      labelTooltip,
       disabled,
       ...other
     } = this.props;
     return (
       <span className="gv-switch-wrapper">
-        {this.renderLabel()}
+        {label ? (
+          labelTooltip ? (
```

Comment:
This is bad.
Try to find solution that will look good and be well supported.

Reply:
Reverted changing in GVSwitch and added new component TooltipLabel which is used with prop of label